python/mlcroissant/mlcroissant/_src/datasets_test.py

```diff
@@ -114,6 +117,46 @@ def load_records_and_test_equality(
     assert len(expected_records) == length
 
 
+def _equal_to_set(expected):
+    """Tests that 2 PCollections are equal as sets."""
+
```

Comment:
I guess this is not a test, maybe "checks whether 2 PCollections ..." ?

Reply:
Done.